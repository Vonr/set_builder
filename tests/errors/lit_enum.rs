use set_builder::set;

fn main() {
    set! { x };
}
