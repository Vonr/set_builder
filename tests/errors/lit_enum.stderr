error: proc macro panicked
 --> tests/errors/lit_enum.rs:4:5
  |
4 |     set! { x };
  |     ^^^^^^^^^^
  |
  = help: message: expected `:` after bindings, if you were trying to create an array, use `[...]` instead
